tests/src/certs.c

```diff
@@ -21,6 +21,14 @@
 
 #include <test/certs.h>
 
+#include "mbedtls/build_info.h"
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
After we decided to add `psa/crypto.h` to `legacy_or_psa.h` in x509 PR I think this can be removed from here.

Also `MBEDTLS_PSA_CRYPTO_C` is more suitable as `MBEDTLS_HAS_ALG_xxxx_VIA_MD_OR_PSA` macros require also `PSA_WANT_ALG_xxxx` if `MBEDTLS_USE_PSA_CRYPTO` is not defined.